src/OrbitGl/App.cpp

```diff
@@ -266,10 +269,28 @@ OrbitApp::~OrbitApp() {
 }
 
 void OrbitApp::OnCaptureFinished(const CaptureFinished& capture_finished) {
-  if (capture_finished.status() == CaptureFinished::kFailed) {
-    SendErrorToUi("Capture Failed", capture_finished.error_message());
-    ERROR("Capture Finished with error: %s", capture_finished.error_message());
-  }
+  main_thread_executor_->Schedule([this, capture_finished]() {
+    switch (capture_finished.status()) {
+      case orbit_grpc_protos::CaptureFinished_Status_kSuccessful: {
```

Comment:
Nit: Using the abbreviated constants like `CaptureFinished::kFailed` etc. would make it more readable. At least to me.

Reply:
Oh yeah, the cases were autogenerated by the IDE and I forgot to change them. Done.